tests/views/test_general.py

```diff
@@ -18,110 +17,106 @@ def test_health_check(client):
     assert resp.status_code == 200
     assert resp.data == "got it!".encode()
 
-    trx = Transaction.objects.get(id=id_trx)
+    trx = Transaction.objects.get(id=outcome_transaction.id)
     assert trx.estado is Estado.failed
 
     num_events = len(trx.events)
-    data = dict(id=DEFAULT_ORDEN_ID, Estado='DEVOLUCION', Detalle="0")
+    data = dict(
+        id=outcome_transaction.stp_id, Estado='DEVOLUCION', Detalle="0"
+    )
     resp = client.post('/orden_events', json=data)
     assert resp.status_code == 200
     assert resp.data == "got it!".encode()
 
-    trx = Transaction.objects.get(id=id_trx)
+    trx = Transaction.objects.get(id=outcome_transaction.id)
     assert trx.estado is Estado.failed
     assert len(trx.events) == num_events
-    trx.delete()
 
-    trx.delete()
 
+@pytest.mark.usefixtures('mock_callback_queue')
+def test_cancelled_transaction(client, outcome_transaction) -> None:
+    data = dict(
+        id=outcome_transaction.stp_id,
+        Estado='CANCELACION',
+        Detalle='something went wrong',
+    )
+    resp = client.post('/orden_events', json=data)
+    assert resp.status_code == 200
+    assert resp.data == "got it!".encode()
+
+    trx = Transaction.objects.get(id=outcome_transaction.id)
+    assert trx.estado is Estado.failed
+    assert trx.detalle == 'something went wrong'
 
-def test_invalid_order_event(client, default_outcome_transaction):
-    trx = Transaction(**default_outcome_transaction)
-    trx.stp_id = DEFAULT_ORDEN_ID
-    trx.save()
-    id_trx = trx.id
 
+def test_invalid_order_event(client, outcome_transaction):
     data = dict(Estado='LIQUIDACION', Detalle="0")
     resp = client.post('/orden_events', json=data)
     assert resp.status_code == 200
     assert resp.data == "got it!".encode()
 
-    trx = Transaction.objects.get(id=id_trx)
+    trx = Transaction.objects.get(id=outcome_transaction.id)
     assert trx.estado is Estado.created
-    trx.delete()
 
 
-def test_invalid_id_order_event(client, default_outcome_transaction):
-    trx = Transaction(**default_outcome_transaction)
-    trx.stp_id = DEFAULT_ORDEN_ID
-    trx.save()
-    id_trx = trx.id
-
+def test_invalid_id_order_event(client, outcome_transaction):
     data = dict(id='9', Estado='LIQUIDACION', Detalle="0")
     resp = client.post('/orden_events', json=data)
     assert resp.status_code == 200
     assert resp.data == "got it!".encode()
 
-    trx = Transaction.objects.get(id=id_trx)
+    trx = Transaction.objects.get(id=outcome_transaction.id)
     assert trx.estado is Estado.created
-    trx.delete()
-
 
-def test_order_event_duplicated(
-    client, default_outcome_transaction, mock_callback_queue
-):
-    trx = Transaction(**default_outcome_transaction)
-    trx.stp_id = DEFAULT_ORDEN_ID
-    trx.save()
-    id_trx = trx.id
 
-    data = dict(id=DEFAULT_ORDEN_ID, Estado='LIQUIDACION', Detalle="0")
+@pytest.mark.usefixtures('mock_callback_queue')
+def test_order_event_duplicated(client, outcome_transaction):
```

Comment:
Creo que si vas a usar `pytest.mark.usefixtures` deberías usarlo para todos los fixtures, como `outcome_transaction`. 
```suggestion
@pytest.mark.usefixtures('mock_callback_queue', 'outcome_transaction')
def test_order_event_duplicated(client):
```

Reply:
no podría usarlo como sugieres porque `outcome_transaction ` si se utiliza en el test mientras que `mock_callback_queue ` no